src/util/Enums.js

```diff
@@ -78,4 +79,12 @@ export default {
   CUSTOM_URI_CHALLENGE: 'CUSTOM_URI',
   UNIVERSAL_LINK_CHALLENGE: 'UNIVERSAL_LINK',
   APP_LINK_CHALLENGE: 'APP_LINK',
+
+  // Initial Flows
+  DEFAULT_FLOW: 'default',
+  PROCEED_WITH_FLOW: 'proceed',
+  LOGIN_FLOW: 'login',
+  REGISTRATION_FLOW: 'signup',
+  RESET_PASSWORD_FLOW: 'resetPassword',
```

Comment:
how did we come up with these unique names for eg `resetPassword` `signup` `login`?

For remediation `login` would be `identify` in idx response...

Reply:
If we believe there will be many of these unique keys, then we should follow some sort of standard to name them.. Ideally they will still be tightly coupled with idx remediation form names to display the correct flow.. right ?

Reply:
There will be an implicit coupling to the idx remediation form names, but these values are purposely more "colloquial" than the remediation names because customer will most likely not be well versed in `idx`

Reply:
@jaredperreault-okta correct the names should be developer friendly and what they can understand.. I was mainly referring to should we use camel case, kebab case ?

Reply:
Ah good point, as JS devs we  just defaulted to camel case. There is a parallel change that has been made in `auth-js` to mimic this same feature which also uses these same strings

Reply:
perfect